docs/Dialects/Seq/RationaleSeq.md

```diff
@@ -119,10 +119,13 @@ from the design.
 reset is asynchronous.
 - **isAsync**: Optional boolean flag indicating whether the reset is
 asynchronous.
+- **preset**: Optional attribute specifying a preset value. If no preset
+attribute is present, the register is random-initialized.
```

Comment:
This is not technically true as the random initialization is guarded under an `ifdef`. I.e., the register is random-initialized if some defines are additionally set. I would consider just striking the second sentence.

Some more notes below on changing the code to emit both random init and initial

Reply:
Here I'd make a distinction between random init in CIRCT and random init in the context of all the SV defines. As far as I'm concerned, if we take the code path of emitting random init, be in under ifdef guard, I consider the register to be random-initialized.